### PR TITLE
Print unhandled exception to stdout depending on verbosity

### PR DESCRIPTION
**Pull request by cblp**

### CODE REVIEW

**Comment by ocramz** [comment]
Thank you @cblp !

File: Web/Scotty/Action.hs [hunk 1]
```diff
@@ -171,9 +174,13 @@ scottyExceptionHandler = Handler $ \case
...
 -- | Uncaught exceptions turn into HTTP 500 Server Error codes
-someExceptionHandler :: MonadIO m => ErrorHandler m
-someExceptionHandler = Handler $ \case
-  (_ :: E.SomeException) -> status status500
+someExceptionHandler :: MonadIO m => Options -> ErrorHandler m
+someExceptionHandler Options{verbose} =
+  Handler $ \(E.SomeException e) -> do
+    when (verbose > 0) $
+      liftIO $
+      putStrLn $ "Caught and exception of " <> show (typeOf e) <> ": " <> show e
```
Comment (fumieval) [inline Web/Scotty/Action.hs:182]: Is it intentional that it goes to stdout, not stderr?

Comment (cblp) [reply Web/Scotty/Action.hs:182]: https://12factor.net/logs recommend stdout, but I'm not sure

Comment (fumieval) [reply Web/Scotty/Action.hs:182]: I don't have a strong opinion; I think stdout is fine too. By the way `Caught and exception of ` looks like a typo

Comment (cblp) [reply Web/Scotty/Action.hs]: Fixed typo

**Review by fumieval — APPROVED** [review]

File: Web/Scotty/Action.hs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE CPP               #-}
+{-# LANGUAGE NamedFieldPuns    #-}
```
Comment (ocramz) [inline Web/Scotty/Action.hs:4]: What is this for?

**Review by ocramz — APPROVED** [review]
Thanks! 👍

File: Web/Scotty/Action.hs (hunk 2 above)
Comment (cblp) [reply Web/Scotty/Action.hs:4]: This allows to read record fields in most natural way (in my taste):
```hs
someExceptionHandler Options{verbose} =
    when (verbose > 0) ...
```

**Comment by ocramz** [comment]
Hi @cblp , there are some small conflicts due to a PR I recently merged. Happy to merge your changes once they are fixed. Thanks again!

**Review by ocramz — CHANGES_REQUESTED** [review]
Could you please add a CHANGELOG entry as well?

**Review by ocramz — APPROVED** [review]

**Comment by cblp** [comment]
Thanks!

**Pull request merged**